Userland/Libraries/LibCore/System.cpp

```diff
@@ -1758,6 +1766,15 @@ ErrorOr<String> current_executable_path()
     auto ret = _NSGetExecutablePath(path, &size);
     if (ret != 0)
         return Error::from_errno(ENAMETOOLONG);
+#elif defined(AK_OS_HAIKU)
+    image_info info;
```

Comment:
This should probably be zero initialized  `image_info info = {};`

Reply:
Thanks, done.